language-agent/Meter.proto

```diff
@@ -30,6 +30,10 @@ service MeterReportService {
     // Meter data is reported in a certain period. The agent/SDK should report all collected metrics in this period through one stream.
     rpc collect (stream MeterData) returns (Commands) {
     }
+
+    // It is recommended to keep the stream alive for performance considerations
```

Comment:
I think this is not a good command for a service.

```suggestion
    // Reporting meter data in bulk mode as MeterDataCollection.
    // By using this, each one in the stream would be treated as a complete input for MAL engine.
    // Comparing to `collect (stream MeterData)`, one stream is an input data set.
```

Reply:
Updated.